vpr/src/place/net_cost_handler.h

```diff
@@ -126,6 +126,16 @@ class NetCostHandler {
      */
     double get_total_wirelength_estimate() const;
 
+    /**
+     * @brief Estimates routing channel utilization.
+     *
+     * For each net, distributes estimated wirelength across its bounding box
+     * and accumulates demand for different routing channels. Normalizes by channel widths.
+     *
+     * @return Pair of matrices with relative CHANX and CHANY utilization.
```

Comment:
List the dimension order here.

Reply:
Normalizes by channel width (e.g. a value of 0.5 means 50% of the wiring in a channel is expected to be used).